modules/openapi-generator/src/test/java/org/openapitools/codegen/erlang/ErlangServerCodegenTest.java

```diff
@@ -0,0 +1,39 @@
+package org.openapitools.codegen.erlang;
+
+import org.openapitools.codegen.ClientOptInput;
+import org.openapitools.codegen.DefaultGenerator;
+import org.openapitools.codegen.config.CodegenConfigurator;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.openapitools.codegen.TestUtils.newTempFolder;
+
+public class ErlangServerCodegenTest {
+
+    @Test
```

Comment:
This I didn't know how to write, don't know any java, so thanks for that! However @wing328 if you have any java guy it'd be good someone else double-checks this for me.

Reply:
👌

will review accordingly